src/material-experimental/mat-feature-highlight/feature-highlight-callout-container.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {BasePortalOutlet, CdkPortalOutlet, ComponentPortal, TemplatePortal} from '@angular/cdk/portal';
+import {Component, ComponentRef, EmbeddedViewRef, OnDestroy, ViewChild} from '@angular/core';
+
+/** Container for the callout component of feature highlight. */
+@Component({
+  selector: 'feature-highlight-callout-container',
+  templateUrl: './feature-highlight-callout-container.ng.html',
+})
+export class FeatureHighlightCalloutContainer extends BasePortalOutlet
+    implements OnDestroy {
+  @ViewChild(CdkPortalOutlet, {static: true}) portalOutlet!: CdkPortalOutlet;
```

Comment:
We should avoid adding new code that depends on static queries. Also you can drop the `!` here since we haven't enabled the compiler flag for it.